DSI/globus_gridftp_server_iRODS.cpp

```diff
@@ -1674,6 +1677,186 @@ globus_l_gfs_iRODS_command(
 
 }
 
+void run_writer_thread(
```

Comment:
The word `run` in this function name makes me think that this is spawning threads when in fact it is the task being posted to the `thread_pool`. Does it make sense to choose a name that communicates this? Maybe it's just me.

Reply:
Maybe execute_writer_thread_operation?

Reply:
Done